events/Shire Bree Union.txt

```diff
@@ -0,0 +1,54 @@
+# Remove this line to have the validator check for strings in localisation
+# Audax Validator "!" Ignore_1004
+
+# Remove this line to have the validator check this file
+# Audax Validator "!" Ignore_NEXT
+
+# Shire-Breeland Union Events: {
+# 60000 - 60099 }
+
+country_event = {
+	id = 60000
+	picture = "map_of_eriador"
+	title = "The Buckland Question"
+	desc = "EVTDESC60000"
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = "We should let Buckland be independent"
+		prestige = -5
+		badboy = -1
+		release_vassal = BCK
+		BCK = {
+			prestige = 5
+		}
+		relation = {
+			who = BCK
+			value = 50
+		}
+		ai_chance = {
+			factor = 3
+		}
+	}
+	
+	option = {
+		name = "We should leave Buckland as it was before the unification - an autonomous colony"
+		create_vassal = BCK
+		ai_chance = {
+			factor = 27
+		}
+	}
+	
+	option = {
+		name = "We should bring Buckland into our fold"
+		prestige = 5
+		badboy = 1
+		inherit = BCK
+		BCK = { all_core = { add_core = SBU} }
+		add_accepted_culture = buckland_hobbit
+		ai_chance = {
+			factor = 70
+		}
+	}

```

Comment:
Parts of this probably shouldn't happen if Buckland is the vassal of another realm, or if Buckland doesn't exist.

You'll want to do some if-emulation to determine if Buckland exists, and handle it differently. e.g.


```
prestige = 5
badboy = 1
BCK = { all_core = { add_core = SBU} }
add_accepted_culture = buckland_hobbit

random_owned = {
    limit = {
        owner = { 
            exists = BCK
            OR = {
                BCK = { is_vassal = no }
                is_our_vassal = BCK
            }
        }
    }
    inherit = BCK
}
```